packages/grid/x-data-grid-pro/src/internals/hooks/features/export/serializer/excelSerializer.ts

```diff
@@ -0,0 +1,181 @@
+import type * as Excel from 'exceljs';
+import {
+  GridCellParams,
+  GridStateColDef,
+  GridRowId,
+  GridColDef,
+  GridValueOptionsParams,
+  GridValueFormatterParams,
+  GridApi,
+} from '@mui/x-data-grid';
+import { GridExceljsProcessInput } from '../gridExcelExportInterface';
+import { buildWarning } from './utils';
+
+const getExcelJs = () => import('exceljs');
+
+const warnInvalidFormattedValue = buildWarning([
+  'MUI: When the value of a field is an object or a `renderCell` is provided, the Excel export might not display the value correctly.',
+  'You can provide a `valueFormatter` with a string representation to be used.',
+]);
+
+const getFormattedValueOptions = (
+  colDef: GridColDef,
+  { id, row, field }: GridValueOptionsParams,
+  api: GridApi,
+) => {
+  if (!colDef.valueOptions) {
+    return [];
+  }
+  let valueOptionsFormatted;
+  if (typeof colDef.valueOptions === 'function') {
+    valueOptionsFormatted = colDef.valueOptions({ id, row, field });
+  } else {
+    valueOptionsFormatted = colDef.valueOptions;
+  }
+
+  if (colDef.valueFormatter) {
+    valueOptionsFormatted = valueOptionsFormatted.map((option) => {
+      if (typeof option === 'object') {
+        return option;
+      }
+
+      const params: GridValueFormatterParams = { field, api, value: option };
+      return String(colDef.valueFormatter!(params));
+    });
+  }
+  return valueOptionsFormatted.map((option) =>
+    typeof option === 'object' ? option.label : option,
+  );
+};
+
+const serialiseRow = (
+  id: GridRowId,
+  columns: GridStateColDef[],
+  getCellParams: (id: GridRowId, field: string) => GridCellParams,
+  api: GridApi,
+) => {
+  const row = {};
+  const dataValidation = {};
+
+  const firstCellParams = getCellParams(id, columns[0].field);
+  const outlineLevel = firstCellParams.rowNode.depth;
+
+  columns.forEach((column) => {
+    const cellParams = getCellParams(id, column.field);
+    switch (cellParams.colDef.type) {
+      case 'singleSelect': {
+        const valueOptions: GridValueOptionsParams = {
+          id: cellParams.id,
+          row: cellParams.row,
+          field: cellParams.field,
+        };
+        const formattedValueOptions = getFormattedValueOptions(
+          cellParams.colDef,
+          valueOptions,
+          api,
+        );
+        dataValidation[column.field] = {
+          type: 'list',
+          allowBlank: true,
+          formulae: [formattedValueOptions.map((x) => `"${x}"`)],
```

Comment:
When I try to open in Excel the file generated in https://deploy-preview-3981--material-ui-x.netlify.app/components/data-grid/export/#excel-export it claims that the validation formula is wrong and it asks if I want to to try to recover the file. In the ExcelJS GitHub page it says that the values should be separated by comma. In my Excel (Portuguese) I need to use semicolon for it to work:

<img width="400" src="https://user-images.githubusercontent.com/42154031/155407757-73775b47-ef10-417d-8a0c-201bee092506.png" alt="image" style="max-width: 100%;">

Reply:
I miss understood the doc.
Instead of using
```ts
formulae: ['"option1,oprion2,option3"']
```
I used 
```ts
formulae: ['"option1","oprion2","option3"']
```

That's corrected. It works better on Google Sheet, on Microsoft, I don't know